devfiles/index.json

```diff
@@ -1,62 +1,62 @@
 [
 {
-  "displayName":"Java MicroProfile template",
-  "description":"Cloud Microservice Starter for Java - MicroProfile / Java EE",
+  "displayName":"WebSphere Liberty MicroProfile®",
+  "description":"Eclipse MicroProfile® on Websphere Liberty",
   "language":"java",
   "projectType":"liberty",
   "location":"https://github.com/microclimate-dev2ops/javaMicroProfileTemplate",
   "links": {"self":"/devfiles/javaMicroProfileTemplate/devfile.yaml" }
 }
 ,{
-  "displayName":"Spring template",
-  "description":"Template for building a Spring microservice",
+  "displayName":"Spring Boot®",
+  "description":"Spring Boot® using IBM Java",
   "language":"java",
   "projectType":"spring",
   "location":"https://github.com/microclimate-dev2ops/springJavaTemplate",
   "links": {"self":"/devfiles/springJavaTemplate/devfile.yaml" }
 }
 ,{
-  "displayName":"Python template",
+  "displayName":"Python",
   "description":"HelloWorld microservice written in python",
   "language":"python",
   "projectType":"docker",
   "location":"https://github.com/microclimate-dev2ops/SVTPythonTemplate",
   "links": {"self":"/devfiles/pythonTemplate/devfile.yaml" }
 }
 ,{
-  "displayName":"Node.js template",
+  "displayName":"Node.js Express",
   "description":"Cloud-ready Node.js Express sample application",
   "language":"nodejs",
   "projectType":"nodejs",
   "location":"https://github.com/microclimate-dev2ops/nodeExpressTemplate",
   "links": {"self":"/devfiles/nodeExpressTemplate/devfile.yaml" }
 }
 ,{
-  "displayName":"Swift microservice template",
-  "description":"Template for building a Swift microservice",
+  "displayName":"Swift",
+  "description":"Sample for building a Swift microservice",
```

Comment:
Compare to the Go description, can we make the format more consistent across different templates? e.g this one is Sample for building xxx microservice but the Go one is Sample microservice for xxx.